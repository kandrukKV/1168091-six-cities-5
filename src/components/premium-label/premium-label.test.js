import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import PremiumLabel from "./premium-label";

it(`Preloader is rendered correctly`, () => {
  const renderer = new ShallowRenderer();
  const tree = renderer.render(
      <PremiumLabel
        className={`className`}
      />
  );
  expect(tree).toMatchSnapshot();
});
