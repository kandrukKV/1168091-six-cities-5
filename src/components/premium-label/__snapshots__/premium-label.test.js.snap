// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader is rendered correctly 1`] = `
<div
  className="className__mark"
>
  <span>
    Premium
  </span>
</div>
`;
